util/uri/src/lib.rs

```diff
@@ -69,7 +71,7 @@ impl UriScheme for ConsensusPeerScheme {
     const DEFAULT_INSECURE_PORT: u16 = 8080;
 }
 
-/// Admin Uri Scheme
+/// Consensus Admin Uri Scheme
```

Comment:
This is not consensus-specific. This is for all MobileCoin services.

Reply:
will fix, sorry